crates/smart-config-commands/examples/cli.rs

```diff
@@ -195,7 +198,8 @@ const JSON: &str = r#"
 const YAML: &str = r#"
 test:
   port: 3000
-  poll_latency_ms: 300
+  poll_latency_in_ms: 300
```

Comment:
As for `in_`, soft -0 on this. IMHO, having one right way to do things is better. Ability to specify either option can cause more issues later, especially when configs are added/edited by people not really familiar with this library.

Reply:
Well, it's a soft -0, so no strong opinion :) 
It feels a bit like tailoring library for bad practices that are already in place (rather than fixing the root cause and standardizing things), but I can certainly see how it could be easier to just support it in the library.